src/win/getdelim.h

```diff
@@ -0,0 +1,30 @@
+/*
+    This file is part of darktable,
+    copyright (c) 2009--2012 johannes hanika.
+    copyright (c) 2010--2012 tobias ellinghaus.
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef __GETDELIM_H__
+#define __GETDELIM_H__
```

Comment:
A word of caution about defines like that, `__*__` define pattern is generally reserved for system headers.
I **really** don't like that even now we are using things like this internally.

```
7.1.3 Reserved identifiers

Each header declares or defines all identifiers listed in its associated subclause, and optionally declares or
defines identifiers listed in its associated future library directions subclause and identifiers which are always
reserved either for any use or for use as file scope identifiers.

- All identifiers that begin with an underscore and either an uppercase letter or another underscore are always
  reserved for any use.
- All identifiers that begin with an underscore are always reserved for use as identifiers with file scope in both
  the ordinary and tag name spaces.
- Each macro name in any of the following subclauses (including the future library directions) is reserved for
  use as specified if any of its associated headers is included; unless explicitly stated otherwise (see 7.1.4).
- All identifiers with external linkage in any of the following subclauses (including the future library directions)
  are always reserved for use as identifiers with external linkage.154
- Each identifier with file scope listed in any of the following subclauses (including the future library directions)
  is reserved for use as a macro name and as an identifier with file scope in the same name space if any of
  its associated headers is included.
```
https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier/228797#228797